invenio_rdm_migrator/streams/cache.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, filepath=None, validate=False):
         """Constructor."""
         self._data = {}
 
-        if filepath:  # load cache from file
+        if filepath and filepath.exists():  # load cache from file
```

Comment:
To reinforce what @alejandromumo mentioned, could we put some `print(f"loading cache from {fliepath}")` statements to know what's happening?

Reply:
done, added also for dumping:

e.g.
```
Loading cache from /Users/ppanero/Workspace/zenodo/demo/cache/parents.json 2023-05-05T11:27:50.212081
Finished loading cache 2023-05-05T11:27:50.212170
Cache loading took 0.00010991096496582031 seconds.
Loading cache from /Users/ppanero/Workspace/zenodo/demo/cache/records.json 2023-05-05T11:27:50.212217
Finished loading cache 2023-05-05T11:27:50.212278
Cache loading took 7.390975952148438e-05 seconds.
Stream started 2023-05-05T11:27:50.212548
Dumping cache to /Users/ppanero/Workspace/zenodo/demo/cache/parents.json 2023-05-05T11:27:50.212678
Finished dumping cache 2023-05-05T11:27:50.213631
Cache dumping took 0.0009357929229736328 seconds.
Dumping cache to /Users/ppanero/Workspace/zenodo/demo/cache/records.json 2023-05-05T11:27:50.213669
Finished dumping cache 2023-05-05T11:27:50.213820
Cache dumping took 0.0001399517059326172 seconds.
```